modules/spring-config/pom.xml

```diff
@@ -69,12 +27,6 @@
             <artifactId>spring-context</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>cglib</groupId>
-            <artifactId>cglib</artifactId>
-            <version>${springCglibVersion}</version>
```

Comment:
remove this property from the root pom if it is not used elsewhere

Reply:
It was declared in this POM on line 17, removed as part of this commit.